addons/website/views/website_templates.xml

```diff
@@ -41,14 +41,16 @@
             'data-website-id': website.id if website else None,
             'data-edit_translations': '1' if edit_translations else None,
         }"/>
-        <t t-if="editable or translatable" t-set="nothing" t-value="html_data.update({
+        <t t-if="env['ir.ui.view'].user_has_groups('base.group_user') and (editable or translatable)" t-set="nothing" t-value="html_data.update({
```

Comment:
**BEFORE MERGE**

@younn-o Is not checking if `editable` enough to not print those? I don't think a new call to `user_has_groups` is required :grimacing:

Reply:
Yes it is not necessary at all... I am removing that